portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,92 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let root = document.documentElement;
+
+const secretText = document.querySelector("#secret-text"); 
+const mePic = document.querySelector("#me-pic"); 
+const tomiePic = document.querySelector("#tomie-pic"); 
+const mainHeading = document.querySelector("#main-heading"); 
+
+//fade in main text
+const mainText = document.querySelector(".main-text");
+window.addEventListener('load', e=> {
+    mainText.classList.add("fade-text-in"); 
+})
+
 /**
- * Adds a random greeting to the page.
+ * Keeps track of mouse and updates our special, circle pointer to
+ * follow cursor movement. 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+const mouseCursor = document.querySelector(".cursor");
```

Comment:
Better naming:  This isn't actually the mouse cursor, it's a circle that follows the mouse cursor.  For someone not familiar with your code this could get confusing.